tests/formatter.rs

```diff
@@ -2448,7 +2448,7 @@ fn secondary_title_no_level_text() {
         )
         .element(
             Level::NOTE
-                .text(None)
+                .text(None::<&str>)
```

Comment:
Is this still needed?

Reply:
sadly yes

Reply:
How come and then why does `OptionCow` exist?  I thought it was to bypass this problem

Reply:
`::<&str>` is needed because the type of `None` can't be inferred, as the `From` impl for `Option` is generic over `T: Into<Cow<'a, str>>`. This problem would still exist if we were to write out the different `From`. 

`OptionCow` exists to solve the more common case of passing in a literal without also needing to write `Some()`.

Reply:
I have a way of solving this with `None` with a new type but I thought I had a way of solving this without `None` without any new types but my ideas aren't working and not finding any breadcrumbs (in clap I went from `impl Into<Option<&str>>` straight to the solution for `None`).